src/test/java/nextstep/cucumber/steps/PathStepDef.java

```diff
@@ -66,7 +68,13 @@ public PathStepDef() {
         When("사용자가 {string}에서 {string}까지의 경로를 조회하면", (String start, String end) -> {
             Long startId = stationIds.getOrDefault(start, -1L);
             Long endId = stationIds.getOrDefault(end, -1L);
-            response = 지하철역_경로조회(startId, endId);
+            response = 지하철역_경로조회(startId, endId, PathType.DISTANCE);
+        });
+
+        When("사용자가 {string}에서 {string}까지의 최소 시간 기준으로 경로를 조회하면", (String start, String end) -> {
+            Long startId = stationIds.getOrDefault(start, 1L);
+            Long endId = stationIds.getOrDefault(end, 3L);
```

Comment:
```suggestion
            var startId = stationIds.getOrDefault(start, 1L);
            var endId = stationIds.getOrDefault(end, 3L);
```

다음에는 타입 추론도 한번 사용해보실 바래요~ 타입 추론을 사용하면 코드가 간결해지고, 변수명이 직관적으로 변한답니다.